### ISSUE TITLE
Val and Test set of iSAID

### ISSUE THREAD

**Issue by xiaollu** [linhanwang/SCCNet#2]
Hi! Thanks for your great work! I am just a bit confused about the setting of the validation set and test set in the iSAID dataset. I noticed that the ground truths of the iSAID test set are unavailable, could you explain a bit about your setting? Thank you very much.

**Comment by linhanwang** [comment]
Sorry for the late response. We only have train and val dataset. I know this is misleading but I follow the naming of iSaid. So the val here is actually test. If I failed to answer your question, could you please provide more information?

**Comment by xiaollu** [comment]
Thanks for your response. It resolved my confusion :)
